prov/util/src/util_ep.c

```diff
@@ -248,7 +248,9 @@ int ofi_endpoint_init(struct fid_domain *domain, const struct util_prov *util_pr
 		ofi_ep_bind_eq(ep, util_domain->eq);
 
 	/* TODO Figure out how to optimize this lock for rdm and msg endpoints */
-	ret = ofi_genlock_init(&ep->lock, OFI_LOCK_MUTEX);
+	ret = ofi_genlock_init(&ep->lock,
```

Comment:
Remove above comment since it is still optimized

Reply:
done